campaigns/mofo-eoy-2016/snippet.html

```diff
@@ -0,0 +1,300 @@
+{#
+2016 fundraising snippet.
+
+Variables:
+
+  @text: main text - Text of snippet.
+  @icon: image - Snippet icon. 64x64px. SVG or PNG preferred.
+  @button_label: Required. Text for donate button.
+  @donation_form_url: Required. Url to the donation form.
+
+  @button_color: Default blue. Red and green also options.
```

Comment:
Note: "red" is a bad option for button color because it's disruptive and also means a dangerous action. I know we default to blue so I won't block on this but please communicate this to folks asking for red buttons. 

It also made ppl upset in the past!